bindings/py/src/nupic/bindings/regions/PyRegion.py

```diff
@@ -404,3 +406,31 @@ def executeMethod(self, methodName, args):
       raise Exception('Command: ' + methodName + ' must be callable')
 
     return m(*args)
+
+  @staticmethod
+  def setSparseOutput(outputs, name, value):
+    """
+    Set region sparse output value.
+
+    The region output memory is owned by the c++ caller and cannot be changed  
+    directly from python. Use this method to update the sparse output fields in  
+    the "outputs" array so it can be resized from the c++ code.
+
+    :param outputs: (dict) of numpy arrays (one per output)
```

Comment:
why would you pass multiple things here? it looks like you can only pass one name and value in the other two params